packages/api-axios/src/disclaimers.js

```diff
@@ -0,0 +1,9 @@
+import axios from 'axios';
+import merge from 'merge-options-es5';
```

Comment:
axios already has a merge utility so no need to pass this in

Reply:
axios's merge utility sucks